core/src/components/item-divider/item-divider.md.scss

```diff
@@ -44,7 +44,10 @@
 ::slotted(ion-icon) {
   color: $item-md-icon-slot-color;
 
-  font-size: $item-md-icon-slot-font-size;
+  // The font size should be 24px which is
+  // calculated in relation to the host's font
+  // size of 14px. e.g. 24 / 14 = 1.714
+  font-size: 1.71em;
```

Comment:
We can't use the `$item-md-icon-slot-font-size` variable here anymore because of the following:

- The value of `$item-md-icon-slot-font-size` before converting to `em` units is `24px`
- **Item** uses a `font-size` of `16px` which converts the value to `1.5em`
- **Item divider** uses a `font-size` of `14px` which, when this value is set to `1.5em`, makes the pixel value equal to 21px
- Therefore, in order to avoid visual diffs we need to specifically set it to `1.71em` for item divider so the converted value is equal to `24px` still

Reply:
Why not use?:

```scss
$item-md-icon-slot-font-size: 24px;
$item-divider-md-font-size-base: 14px;

font-size: math.div($item-md-icon-slot-font-size, $item-divider-md-font-size-base) * 1em;
```

This would let us keep the variables and make it easy for us to understand the chosen values without the need of comments.

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/27989/commits/9cb4f2739103706f12449dc450c36dc5f8e26db3